proto/buf.yaml

```
@@ -1,7 +1,7 @@
 version: v1
 name: buf.build/axone-protocol/axoned
 deps:
-  - buf.build/cosmos/cosmos-sdk:v0.50.1
+  - buf.build/cosmos/cosmos-sdk:v0.50.4
```

```suggestion
  - buf.build/cosmos/cosmos-sdk:v0.50.6
```

---

```suggestion
  - buf.build/cosmos/cosmos-sdk:v0.50.0
```

Good catch, but by trying this and update buf dependencies (`buf dep update`), I noticed that both 0.50.4 and 0.50.6 do not work because only 0.50.0 has been published on buf. So I put it back to 0.50.0.

---

Yes good :)